src/main/java/edu/cornell/kfs/concur/batch/businessobject/ConcurStandardAccountingExtractDetailLine.java

```diff
@@ -205,6 +216,22 @@ public void setReportEndDate(Date reportEndDate) {
         this.reportEndDate = reportEndDate;
     }
 
+    public String getPolicy() {
+        return policy;
+    }
+
+    public void setPolicy(String policy) {
+        this.policy = policy;
+    }
+
+    public String getExpenseType() {
+        return expenseType;
+    }
+
+    public void setExpenseType(String expenseType) {
+        this.expenseType = expenseType;
+    }
+
     public String getDebugInformation() {
         StringBuilder sb = new StringBuilder(" batchID: ").append(batchID).append(" batchDate: ").append(batchDate);
```

Comment:
The policy and expenseType are being added to the output string; see the changed lines later on in this method.

As per my earlier comment, though, I had a question as to whether these attributes were only added for logging, or if there's supposed to be some code in this PR that will actually use them for PDP processing.